util/mods.py

```diff
@@ -309,39 +432,11 @@ def get_steam_db_rules(mods: Dict[str, Any]) -> Dict[str, Any]:
                     logger.debug(
                         "Returning db.json, this data is long so we forego logging it here"
                     )
-                    return db_data["database"]
+                    db_json_data = db_data["database"]
```

Comment:
If we `break` after this, then the community rules will not be loaded. The loop is broken after the `return db_json_data, community_rules_json_data` and it should not loop through any additional mods, right?

Another thing I have suggested in the past several times - we could also just allow the user to configure the "database mod" themselves, as an alternative to hardcoding RimPy Mod Manager Database directly. The user could then configure the direct path to their preferred "database mod", and we could wrap that function and do warning/error handling if the mod doesn't exist or if there is an issue parsing the .json, etc. This could be calculated earlier where we are already running & caching those more expensive calculations anyways... we load `game_configuration` before mods are even parsed, so if we have that preference predefined, and if the mod is detected as we are parsing the metadata, we could retrieve the paths to the `.json` before we even get to this stage and cache it for later usage in this block.